SwrveSDK/SwrveObject.brs

```diff
@@ -1,211 +0,0 @@
-' brief:      Object fields manipulation helper.
```

Comment:
To simplify thing this class was removed and replaced with a simple helper `SWGetValue` as the only function still being used was the `default` function.